docs/guide/setup.md

```diff
@@ -28,3 +28,24 @@ auth: {
 ::: warning IMPORTANT
 When adding `auth-module` to a new Nuxt project ensure you have activated the Vuex store. More information on how to do that can be found on the [Nuxt Getting Started Guide](https://nuxtjs.org/guide/vuex-store).
 :::
+
+
+### Typescript setup
+
+Add the types to your "types" array in tsconfig.json after the `@nuxt/vue-app` entry
```

Comment:
Since Nuxt 2.9 it's `@nuxt/types` (https://typescript.nuxtjs.org/migration.html)
```suggestion
Add the types to your "types" array in tsconfig.json after the `@nuxt/types` entry
```